src/s1reader/s1_burst_slc.py

```diff
@@ -508,16 +522,29 @@ def _steps_to_vecs(self, range_step, az_step):
         # container to store names of axis vectors that are invalid: i.e. size 0
         vec_errs = []
 
+        rdrgrid = self.as_isce3_radargrid()
+
+        # compute number of margin pixels
+        n_rng_margin = range_margin/self.range_pixel_spacing
+        n_az_margin = az_margin/self.azimuth_time_interval
+
+        # adjust the width and length with margin on both sides
+        width = self.width + 2*n_rng_margin
+        length = self.length + 2*n_az_margin
```

Comment:
```suggestion
        width = self.width + 2 * n_rng_margin
        length = self.length + 2 * n_az_margin
```
Formatting nit for readability